lib/liquid/template.rb

```diff
@@ -183,20 +183,15 @@ def render(*args)
       when Hash
         options = args.pop
 
-        if options[:registers].is_a?(Hash)
-          registers.merge!(options[:registers])
-        end
+        registers.merge!(options[:registers]) if options[:registers].is_a?(Hash)
 
-        if options[:filters]
-          context.add_filters(options[:filters])
-        end
+        context.add_filters(options[:filters]) if options[:filters]
 
-        if options[:exception_handler]
-          context.exception_handler = options[:exception_handler]
-        end
-      when Module
-        context.add_filters(args.pop)
-      when Array
+        context.add_global_filters(options[:global_filters]) if options[:global_filters]
```

Comment:
`default_filters` sounds like it would only be used if there aren't any other filters for that variable.

I considered something like `after_filter` but was concerned about the inverse, it not applying when there aren't any filters. It also sound like it could be applied after every filter.  It doesn't have the advantage of making it clear that it is applied after the other filters rather, which is ambiguous with `auto_filter`.